src/app/src/components/SearchInput.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {
+    Box,
+    Text,
+    Input,
+    InputGroup,
+    InputLeftElement,
+} from '@chakra-ui/react';
+import { IoIosSearch } from 'react-icons/io';
+
+const SearchInput = ({ query, setQuery }) => {
```

Comment:
Could we expose a `placeholder` prop here to that instantiators could set a placeholder title values?